features/post-meta.feature

```diff
@@ -488,3 +488,46 @@ Feature: Manage post custom fields
       """
       [ "new", "bar" ]
       """
+
+  Scenario: Clean up duplicate post meta values.
+    Given a WP install
+    And a session_no file:
+      """
+      n
+      """
+    And a session_yes file:
+      """
+      y
+      """
+
+    When I run `wp post meta add 1 foo bar`
+    Then STDOUT should be:
+      """
+      Success: Added custom field.
+      """
+
+    When I try the previous command again
+    Then the return code should be 0
+
+    When I try the previous command again
+    Then the return code should be 0
+
+    When I run `wp post meta clean-duplicates 1 foo < session_no`
+    # Check for contains only, as the string contains a trailing space.
+    Then STDOUT should contain:
+      """
+      Are you sure you want to delete 2 duplicate enclosures and keep 1 valid enclosures? [y/n]
+      """
+
```

Comment:
Can we add an assertion here to verify there are still three meta keys?

Reply:
Sorry, I didn't get this.

Reply:
Something like:

```
When I run `wp post meta list 1 --keys=foo --format=count`
Then STDOUT should be:
  """
  3
  """
```

Reply:
Done.